src/Sulu/Bundle/AdminBundle/Admin/View/Badge.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Admin\View;
+
+class Badge
+{
+    /**
+     * @var string
+     */
+    private $routeName;
+
+    /**
+     * @var string|null
+     */
+    private $dataPath;
+
+    /**
+     * @var string|null
+     */
+    private $visibleCondition;
+
+    /**
+     * @var array<string, mixed>
+     */
+    private $attributesToRequest = [];
+
+    /**
+     * @var array<string, mixed>
+     */
+    private $routerAttributesToRequest = [];
+
+    public function __construct(string $routeName, ?string $dataPath = null)
```

Comment:
whats the reason for not adding the `visibleCondition` to the constructor?

Reply:
Just forgot about it